src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -34,58 +33,87 @@ import {
   EuiComboBox,
   EuiSpacer,
   EuiFormRow,
-  EuiCode,
-  EuiTextArea,
+  EuiFieldNumber,
 } from '@elastic/eui';
 import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { MODEL_TYPES } from '../../../common/model_options';
+
+const DEFAULTS = {
+  model_type: MODEL_TYPES.UNWEIGHTED,
+  alpha: 0,
+  beta: 0,
+  gamma: 0,
+  period: 1,
+  multiplicative: true,
+  window: 10,
+};
 
 const MovingAverageAggUi = props => {
   const { siblings, intl } = props;
-  const defaults = {
-    settings: '',
-    minimize: 0,
-    window: '',
-    model: 'simple'
-  };
-  const model = { ...defaults, ...props.model };
-  const handleChange = createChangeHandler(props.onChange, model);
-  const handleSelectChange = createSelectHandler(handleChange);
-  const handleTextChange = createTextHandler(handleChange);
-  const handleNumberChange = createNumberHandler(handleChange);
+
+  const model = { ...DEFAULTS, ...props.model };
   const modelOptions = [
     {
-      label: intl.formatMessage({ id: 'tsvb.movingAverage.modelOptions.simpleLabel', defaultMessage: 'Simple' }),
-      value: 'simple'
+      label: intl.formatMessage({
```

Comment:
intl.formatMessage is an obsolete method for localization. Please use i18n.translate instead 

@azasypkin @timroes  Am I right?

Reply:
Replaced

Reply:
That's correct, better use `i18n.translate` where possible (sorry for the late reply).